spec/prog/vm/nexus_spec.rb

```diff
@@ -658,22 +658,30 @@ def new_host(**args)
   end
 
   describe "#wait_sshable" do
-    it "naps if not sshable" do
-      expect(vm).to receive(:ephemeral_net4).and_return("10.0.0.1")
-      expect(Socket).to receive(:tcp).with("10.0.0.1", 22, connect_timeout: 1).and_raise Errno::ECONNREFUSED
+    let(:vm_host) {
+      sshable = instance_double(Sshable)
+      instance_double(VmHost, sshable: sshable)
+    }
```

Comment:
nit:
```suggestion
    let(:vm_host) { instance_double(VmHost, sshable: instance_double(Sshable)) }
```

Reply:
another option: you can move sshable to separate variable and use it as `expect(sshable)` instead of `expect(vm.vm_host.sshable)` to make lines shorter.
```suggestion
    let(:vm_host) { instance_double(VmHost, sshable: sshable) }
    let(:sshable) { instance_double(Sshable) }
```